lib/ansible/module_utils/urls.py

```diff
@@ -1051,7 +1051,7 @@ class Request:
     def __init__(self, headers=None, use_proxy=True, force=False, timeout=10, validate_certs=True,
                  url_username=None, url_password=None, http_agent=None, force_basic_auth=False,
                  follow_redirects='urllib2', client_cert=None, client_key=None, cookies=None, unix_socket=None,
-                 ca_path=None):
+                 ca_path=None, module=None):
```

Comment:
I still do not believe this should be here.  This class should not have any knowledge of modules.  I realize the alternative is insanely more complex, but that doesn't mean that this is the right choice due to ease.

Reply:
Yeah, I agree the composition there is bad.

I like the idea of ripping out the logging. Haven't found yet where it all gets put together. As in... where the `AnsibleModule` is created.